tst/CTA.WebForms2Blazor.Tests/FileConverters/ViewFileConverterTests.cs

```diff
@@ -364,7 +364,49 @@ public async Task TestViewFileConverter_Returns_ContentNode_As_Div()
             Assert.AreEqual(expectedContents, fileContents);
             Assert.AreEqual(relativePath, fi.RelativePath);
         }
+        
+        [Test]
+        public async Task TestViewFileConverter_Converts_Directives()
+        {
+            FileConverter fc = new ViewFileConverter(FileConverterSetupFixture.TestProjectPath, 
+                FileConverterSetupFixture.TestDirectiveFilePath,
+                new ViewImportService(), new TaskManagerService());
+            
+            IEnumerable<FileInformation> fileList = await fc.MigrateFileAsync();
+            FileInformation fi = fileList.Single();
+            
+            byte[] bytes = fi.FileBytes;
+            var fileContents = Encoding.UTF8.GetString(bytes);
+            
+            string newPath = Path.Combine(FileConverterSetupFixture.TestFilesDirectoryPath, "DirectiveOnly.razor");
+            string relativePath = Path.GetRelativePath(FileConverterSetupFixture.TestProjectPath, newPath);
+            relativePath = Path.Combine("Pages", relativePath);
 
+            string expectedContents =  @"@using CTA.WebForms2Blazor.Tests.CustomControls
+@using eShopOnBlazor
+@using Cannot convert file name to namespace, file path Footer.ascx does not have a directory
+@page ""/TestingArea/TestFiles/DirectiveOnly""
+@layout Site
+@inherits eShopLegacyWebForms._Default
+<!-- Conversion of Title attribute (value: ""Home Page"") for Page directive not currently supported -->
+<!-- Conversion of AutoEventWireup attribute (value: ""true"") for Page directive not currently supported -->
```

Comment:
It looks like the original directive is being overwritten by comments. In these cases, can we keep the original node (commented out) and post the new comment(s) above it? 

That way when something isn't converted, the original node can be easily referenced.